src/event.js

```diff
@@ -5,10 +5,11 @@ define( [
 	"./var/rnothtmlwhite",
 	"./var/slice",
 	"./data/var/dataPriv",
+	"./manipulation/var/rcheckableType",
```

Comment:
`rcheckableType` is no longer just for manipulation and should be promoted to a top-level var.